### DISCUSSION TITLE
Framework specific entrypoints

### DISCUSSION THREAD

**Discussion by Malix-Labs** [wxt-dev/wxt#1094] [Q&A] [answered]
About

> | Features | WXT | Plasmo |
> |--------|--------|--------|
> | Framework specific entrypoints (like `Popup.tsx`) | 🟡\[^4] | ✅\[^5] | 
>
> \[^4]: `.html` `.ts` `.tsx`.
> \[^5]: `.html` `.ts` `.tsx`. `.vue` `.svelte`.

\- https://wxt.dev/guide/resources/compare.html#:~:text=Framework%20specific%20entrypoints,%E2%9C%85%205

Are there any plan to add `.vue` and `.svelte` to WXT ?

**Comment by aklinker1** [comment, marked as answer]
No. For HTML files, I always prefer having full control of the HTML file and app creation process, so I don't plan on adding support. So you won't be able to do like `popup.tsx`, `popup.vue`, or `popup.svelte`.

TSX is only supported for Content Scripts, not full HTML pages. Vue and Svelte need a regular TS file for creating and mounting the app, so there's no need to add support for content scripts.

**Reply by Malix-Labs** [reply]
Okay, thanks for your answer !
